### ISSUE TITLE
Incompatible with wagtail 5.0.2

### ISSUE THREAD

**Issue by mcosti** [infoportugal/wagtail-modeltranslation#389]
This is the error I get on accessing any type of page. Doesn't reproduce in wagtail 5.0.1

```python
Traceback (most recent call last):
2023-07-04T08:20:06.565333468Z   File "/usr/local/lib/python3.11/dist-packages/django/forms/forms.py", line 178, in __getitem__
2023-07-04T08:20:06.565336010Z     field = self.fields[name]
2023-07-04T08:20:06.565337885Z             ^^^^^^^^^^^^^^^^^
2023-07-04T08:20:06.565339718Z KeyError: 'slug'
2023-07-04T08:20:06.565341343Z 
2023-07-04T08:20:06.565342968Z During handling of the above exception, another exception occurred:
2023-07-04T08:20:06.565344718Z 
2023-07-04T08:20:06.565346260Z Traceback (most recent call last):
2023-07-04T08:20:06.565347885Z   File "/usr/local/lib/python3.11/dist-packages/django/contrib/staticfiles/handlers.py", line 80, in __call__
2023-07-04T08:20:06.565349677Z     return self.application(environ, start_response)
2023-07-04T08:20:06.565351302Z            ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
2023-07-04T08:20:06.565353260Z   File "/usr/local/lib/python3.11/dist-packages/django/core/handlers/wsgi.py", line 124, in __call__
2023-07-04T08:20:06.565355052Z     response = self.get_response(request)
2023-07-04T08:20:06.565356677Z                ^^^^^^^^^^^^^^^^^^^^^^^^^^
2023-07-04T08:20:06.565358843Z   File "/usr/local/lib/python3.11/dist-packages/django/core/handlers/base.py", line 140, in get_response
2023-07-04T08:20:06.565394385Z     response = self._middleware_chain(request)
2023-07-04T08:20:06.565396760Z                ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
2023-07-04T08:20:06.565398427Z   File "/usr/local/lib/python3.11/dist-packages/django/core/handlers/exception.py", line 57, in inner
2023-07-04T08:20:06.565400177Z     response = response_for_exception(request, exc)
2023-07-04T08:20:06.565401968Z                ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
2023-07-04T08:20:06.565403635Z   File "/usr/local/lib/python3.11/dist-packages/django/core/handlers/exception.py", line 140, in response_for_exception
2023-07-04T08:20:06.565405385Z     response = handle_uncaught_exception(
2023-07-04T08:20:06.565406968Z                ^^^^^^^^^^^^^^^^^^^^^^^^^^
2023-07-04T08:20:06.565408552Z   File "/usr/local/lib/python3.11/dist-packages/django/core/handlers/exception.py", line 181, in handle_uncaught_exception
2023-07-04T08:20:06.565460052Z     return debug.technical_500_response(request, *exc_info)
2023-07-04T08:20:06.565467343Z            ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
2023-07-04T08:20:06.565469302Z   File "/usr/local/lib/python3.11/dist-packages/django_extensions/management/technical_response.py", line 40, in null_technical_500_response
2023-07-04T08:20:06.565471260Z     raise exc_value.with_traceback(tb)
2023-07-04T08:20:06.565473052Z   File "/usr/local/lib/python3.11/dist-packages/django/core/handlers/exception.py", line 55, in inner
2023-07-04T08:20:06.565491093Z     response = get_response(request)
2023-07-04T08:20:06.565494093Z                ^^^^^^^^^^^^^^^^^^^^^
2023-07-04T08:20:06.565496802Z   File "/usr/local/lib/python3.11/dist-packages/django/core/handlers/base.py", line 220, in _get_response
2023-07-04T08:20:06.565498718Z     response = response.render()
2023-07-04T08:20:06.565500343Z                ^^^^^^^^^^^^^^^^^
2023-07-04T08:20:06.565502010Z   File "/usr/local/lib/python3.11/dist-packages/wagtail/admin/auth.py", line 203, in overridden_render
2023-07-04T08:20:06.565504010Z     return render()
2023-07-04T08:20:06.565524343Z            ^^^^^^^^
2023-07-04T08:20:06.565531677Z   File "/usr/local/lib/python3.11/dist-packages/django/template/response.py", line 114, in render
2023-07-04T08:20:06.565535302Z     self.content = self.rendered_content
2023-07-04T08:20:06.565536927Z                    ^^^^^^^^^^^^^^^^^^^^^
2023-07-04T08:20:06.565538552Z   File "/usr/local/lib/python3.11/dist-packages/django/template/response.py", line 92, in rendered_content
2023-07-04T08:20:06.565540718Z     return template.render(context, self._request)
2023-07-04T08:20:06.565542927Z            ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
2023-07-04T08:20:06.565546843Z   File "/usr/local/lib/python3.11/dist-packages/django/template/backends/django.py", line 61, in render
2023-07-04T08:20:06.565553343Z     return self.template.render(context)
2023-07-04T08:20:06.565555927Z            ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
2023-07-04T08:20:06.565557510Z   File "/usr/local/lib/python3.11/dist-packages/django/template/base.py", line 175, in render
2023-07-04T08:20:06.565559343Z     return self._render(context)
2023-07-04T08:20:06.565561093Z            ^^^^^^^^^^^^^^^^^^^^^
2023-07-04T08:20:06.565562677Z   File "/usr/local/lib/python3.11/dist-packages/django/template/base.py", line 167, in _render
2023-07-04T08:20:06.565564593Z     return self.nodelist.render(context)
2023-07-04T08:20:06.565566302Z            ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
2023-07-04T08:20:06.565567927Z   File "/usr/local/lib/python3.11/dist-packages/django/template/base.py", line 1005, in render
2023-07-04T08:20:06.565570093Z     return SafeString("".join([node.render_annotated(context) for node in self]))
2023-07-04T08:20:06.565572802Z                               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
2023-07-04T08:20:06.565577302Z   File "/usr/local/lib/python3.11/dist-packages/django/template/base.py", line 1005, in <listcomp>
2023-07-04T08:20:06.565579177Z     return SafeString("".join([node.render_annotated(context) for node in self]))
2023-07-04T08:20:06.565580843Z                                ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
2023-07-04T08:20:06.565582718Z   File "/usr/local/lib/python3.11/dist-packages/django/template/base.py", line 966, in render_annotated
2023-07-04T08:20:06.565596677Z     return self.render(context)
2023-07-04T08:20:06.565600260Z            ^^^^^^^^^^^^^^^^^^^^
2023-07-04T08:20:06.565601968Z   File "/usr/local/lib/python3.11/dist-packages/django/template/loader_tags.py", line 157, in render
2023-07-04T08:20:06.565603593Z     return compiled_parent._render(context)
2023-07-04T08:20:06.565605302Z            ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
2023-07-04T08:20:06.565618302Z   File "/usr/local/lib/python3.11/dist-packages/django/template/base.py", line 167, in _render
2023-07-04T08:20:06.565629052Z     return self.nodelist.render(context)
2023-07-04T08:20:06.565631718Z            ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
2023-07-04T08:20:06.565634718Z   File "/usr/local/lib/python3.11/dist-packages/django/template/base.py", line 1005, in render
2023-07-04T08:20:06.565640260Z     return SafeString("".join([node.render_annotated(context) for node in self]))
2023-07-04T08:20:06.565643010Z                               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
2023-07-04T08:20:06.565644927Z   File "/usr/local/lib/python3.11/dist-packages/django/template/base.py", line 1005, in <listcomp>
2023-07-04T08:20:06.565646843Z     return SafeString("".join([node.render_annotated(context) for node in self]))
2023-07-04T08:20:06.565648552Z                                ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
2023-07-04T08:20:06.565650260Z   File "/usr/local/lib/python3.11/dist-packages/django/template/base.py", line 966, in render_annotated
2023-07-04T08:20:06.565652052Z     return self.render(context)
2023-07-04T08:20:06.565653843Z            ^^^^^^^^^^^^^^^^^^^^
2023-07-04T08:20:06.565655510Z   File "/usr/local/lib/python3.11/dist-packages/django/template/loader_tags.py", line 157, in render
2023-07-04T08:20:06.565657343Z     return compiled_parent._render(context)
2023-07-04T08:20:06.565658968Z            ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
2023-07-04T08:20:06.565660593Z   File "/usr/local/lib/python3.11/dist-packages/django/template/base.py", line 167, in _render
2023-07-04T08:20:06.565662218Z     return self.nodelist.render(context)
2023-07-04T08:20:06.565664343Z            ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
2023-07-04T08:20:06.565665968Z   File "/usr/local/lib/python3.11/dist-packages/django/template/base.py", line 1005, in render
2023-07-04T08:20:06.565667760Z     return SafeString("".join([node.render_annotated(context) for node in self]))
2023-07-04T08:20:06.565671177Z                               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
2023-07-04T08:20:06.565675593Z   File "/usr/local/lib/python3.11/dist-packages/django/template/base.py", line 1005, in <listcomp>
2023-07-04T08:20:06.565677468Z     return SafeString("".join([node.render_annotated(context) for node in self]))
2023-07-04T08:20:06.565711552Z                                ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
2023-07-04T08:20:06.565720052Z   File "/usr/local/lib/python3.11/dist-packages/django/template/base.py", line 966, in render_annotated
2023-07-04T08:20:06.565722343Z     return self.render(context)
2023-07-04T08:20:06.565723968Z            ^^^^^^^^^^^^^^^^^^^^
2023-07-04T08:20:06.565727302Z   File "/usr/local/lib/python3.11/dist-packages/django/template/loader_tags.py", line 157, in render
2023-07-04T08:20:06.565729177Z     return compiled_parent._render(context)
2023-07-04T08:20:06.565730885Z            ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
2023-07-04T08:20:06.565732510Z   File "/usr/local/lib/python3.11/dist-packages/django/template/base.py", line 167, in _render
2023-07-04T08:20:06.565734260Z     return self.nodelist.render(context)
2023-07-04T08:20:06.565735927Z            ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
2023-07-04T08:20:06.565737552Z   File "/usr/local/lib/python3.11/dist-packages/django/template/base.py", line 1005, in render
2023-07-04T08:20:06.565739260Z     return SafeString("".join([node.render_annotated(context) for node in self]))
2023-07-04T08:20:06.565741010Z                               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
2023-07-04T08:20:06.565742760Z   File "/usr/local/lib/python3.11/dist-packages/django/template/base.py", line 1005, in <listcomp>
2023-07-04T08:20:06.565745218Z     return SafeString("".join([node.render_annotated(context) for node in self]))
2023-07-04T08:20:06.565746927Z                                ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
2023-07-04T08:20:06.565748635Z   File "/usr/local/lib/python3.11/dist-packages/django/template/base.py", line 966, in render_annotated
2023-07-04T08:20:06.565751718Z     return self.render(context)
2023-07-04T08:20:06.565753468Z            ^^^^^^^^^^^^^^^^^^^^
2023-07-04T08:20:06.565755052Z   File "/usr/local/lib/python3.11/dist-packages/django/template/loader_tags.py", line 157, in render
2023-07-04T08:20:06.565756843Z     return compiled_parent._render(context)
2023-07-04T08:20:06.565759093Z            ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
2023-07-04T08:20:06.565760885Z   File "/usr/local/lib/python3.11/dist-packages/django/template/base.py", line 167, in _render
2023-07-04T08:20:06.565762843Z     return self.nodelist.render(context)
2023-07-04T08:20:06.565764343Z            ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
2023-07-04T08:20:06.565765968Z   File "/usr/local/lib/python3.11/dist-packages/django/template/base.py", line 1005, in render
2023-07-04T08:20:06.565767677Z     return SafeString("".join([node.render_annotated(context) for node in self]))
2023-07-04T08:20:06.565769427Z                               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
2023-07-04T08:20:06.565771135Z   File "/usr/local/lib/python3.11/dist-packages/django/template/base.py", line 1005, in <listcomp>
2023-07-04T08:20:06.565780968Z     return SafeString("".join([node.render_annotated(context) for node in self]))
2023-07-04T08:20:06.565783177Z                                ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
2023-07-04T08:20:06.565785052Z   File "/usr/local/lib/python3.11/dist-packages/django/template/base.py", line 966, in render_annotated
2023-07-04T08:20:06.565786927Z     return self.render(context)
2023-07-04T08:20:06.565788552Z            ^^^^^^^^^^^^^^^^^^^^
2023-07-04T08:20:06.565790302Z   File "/usr/local/lib/python3.11/dist-packages/django/template/loader_tags.py", line 63, in render
2023-07-04T08:20:06.565792052Z     result = block.nodelist.render(context)
2023-07-04T08:20:06.565794218Z              ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
2023-07-04T08:20:06.565795802Z   File "/usr/local/lib/python3.11/dist-packages/django/template/base.py", line 1005, in render
2023-07-04T08:20:06.565797468Z     return SafeString("".join([node.render_annotated(context) for node in self]))
2023-07-04T08:20:06.565799177Z                               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
2023-07-04T08:20:06.565800843Z   File "/usr/local/lib/python3.11/dist-packages/django/template/base.py", line 1005, in <listcomp>
2023-07-04T08:20:06.565802468Z     return SafeString("".join([node.render_annotated(context) for node in self]))
2023-07-04T08:20:06.565804135Z                                ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
2023-07-04T08:20:06.565805802Z   File "/usr/local/lib/python3.11/dist-packages/django/template/base.py", line 966, in render_annotated
2023-07-04T08:20:06.565807677Z     return self.render(context)
2023-07-04T08:20:06.565809552Z            ^^^^^^^^^^^^^^^^^^^^
2023-07-04T08:20:06.565811552Z   File "/usr/local/lib/python3.11/dist-packages/django/template/loader_tags.py", line 63, in render
2023-07-04T08:20:06.565813302Z     result = block.nodelist.render(context)
2023-07-04T08:20:06.565815385Z              ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
2023-07-04T08:20:06.565816968Z   File "/usr/local/lib/python3.11/dist-packages/django/template/base.py", line 1005, in render
2023-07-04T08:20:06.565818718Z     return SafeString("".join([node.render_annotated(context) for node in self]))
2023-07-04T08:20:06.565820468Z                               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
2023-07-04T08:20:06.565822260Z   File "/usr/local/lib/python3.11/dist-packages/django/template/base.py", line 1005, in <listcomp>
2023-07-04T08:20:06.565824010Z     return SafeString("".join([node.render_annotated(context) for node in self]))
2023-07-04T08:20:06.565825635Z                                ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
2023-07-04T08:20:06.565827302Z   File "/usr/local/lib/python3.11/dist-packages/django/template/base.py", line 966, in render_annotated
2023-07-04T08:20:06.565833302Z     return self.render(context)
2023-07-04T08:20:06.565835093Z            ^^^^^^^^^^^^^^^^^^^^
2023-07-04T08:20:06.565836677Z   File "/usr/local/lib/python3.11/dist-packages/django/template/loader_tags.py", line 63, in render
2023-07-04T08:20:06.565838427Z     result = block.nodelist.render(context)
2023-07-04T08:20:06.565840177Z              ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
2023-07-04T08:20:06.565841885Z   File "/usr/local/lib/python3.11/dist-packages/django/template/base.py", line 1005, in render
2023-07-04T08:20:06.565844677Z     return SafeString("".join([node.render_annotated(context) for node in self]))
2023-07-04T08:20:06.565866843Z                               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
2023-07-04T08:20:06.565868552Z   File "/usr/local/lib/python3.11/dist-packages/django/template/base.py", line 1005, in <listcomp>
2023-07-04T08:20:06.565870343Z     return SafeString("".join([node.render_annotated(context) for node in self]))
2023-07-04T08:20:06.565873802Z                                ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
2023-07-04T08:20:06.565876927Z   File "/usr/local/lib/python3.11/dist-packages/django/template/base.py", line 966, in render_annotated
2023-07-04T08:20:06.565880927Z     return self.render(context)
2023-07-04T08:20:06.565882843Z            ^^^^^^^^^^^^^^^^^^^^
2023-07-04T08:20:06.565884385Z   File "/usr/local/lib/python3.11/dist-packages/django/template/base.py", line 1064, in render
2023-07-04T08:20:06.565886093Z     output = self.filter_expression.resolve(context)
2023-07-04T08:20:06.565887718Z              ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
2023-07-04T08:20:06.565889343Z   File "/usr/local/lib/python3.11/dist-packages/django/template/base.py", line 715, in resolve
2023-07-04T08:20:06.565891427Z     obj = self.var.resolve(context)
2023-07-04T08:20:06.565893135Z           ^^^^^^^^^^^^^^^^^^^^^^^^^
2023-07-04T08:20:06.565896135Z   File "/usr/local/lib/python3.11/dist-packages/django/template/base.py", line 847, in resolve
2023-07-04T08:20:06.565900260Z     value = self._resolve_lookup(context)
2023-07-04T08:20:06.565901885Z             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
2023-07-04T08:20:06.565903510Z   File "/usr/local/lib/python3.11/dist-packages/django/template/base.py", line 914, in _resolve_lookup
2023-07-04T08:20:06.565905218Z     current = current()
2023-07-04T08:20:06.565906802Z               ^^^^^^^^^
2023-07-04T08:20:06.565908302Z   File "/usr/local/lib/python3.11/dist-packages/wagtail/admin/panels/base.py", line 284, in render_form_content
2023-07-04T08:20:06.565910052Z     return mark_safe(self.render_html() + self.render_missing_fields())
2023-07-04T08:20:06.565911927Z                      ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
2023-07-04T08:20:06.565913552Z   File "/usr/local/lib/python3.11/dist-packages/wagtail/admin/ui/components.py", line 20, in render_html
2023-07-04T08:20:06.565970218Z     return template.render(context_data)
2023-07-04T08:20:06.565975927Z            ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
2023-07-04T08:20:06.565977760Z   File "/usr/local/lib/python3.11/dist-packages/django/template/backends/django.py", line 61, in render
2023-07-04T08:20:06.565979593Z     return self.template.render(context)
2023-07-04T08:20:06.565981260Z            ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
2023-07-04T08:20:06.565986260Z   File "/usr/local/lib/python3.11/dist-packages/django/template/base.py", line 175, in render
2023-07-04T08:20:06.565991802Z     return self._render(context)
2023-07-04T08:20:06.565994010Z            ^^^^^^^^^^^^^^^^^^^^^
2023-07-04T08:20:06.566019510Z   File "/usr/local/lib/python3.11/dist-packages/django/template/base.py", line 167, in _render
2023-07-04T08:20:06.566023843Z     return self.nodelist.render(context)
2023-07-04T08:20:06.566025677Z            ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
2023-07-04T08:20:06.566027343Z   File "/usr/local/lib/python3.11/dist-packages/django/template/base.py", line 1005, in render
2023-07-04T08:20:06.566029052Z     return SafeString("".join([node.render_annotated(context) for node in self]))
2023-07-04T08:20:06.566030802Z                               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
2023-07-04T08:20:06.566032843Z   File "/usr/local/lib/python3.11/dist-packages/django/template/base.py", line 1005, in <listcomp>
2023-07-04T08:20:06.566034677Z     return SafeString("".join([node.render_annotated(context) for node in self]))
2023-07-04T08:20:06.566036385Z                                ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
2023-07-04T08:20:06.566040302Z   File "/usr/local/lib/python3.11/dist-packages/django/template/base.py", line 966, in render_annotated
2023-07-04T08:20:06.566043677Z     return self.render(context)
2023-07-04T08:20:06.566045510Z            ^^^^^^^^^^^^^^^^^^^^
2023-07-04T08:20:06.566047093Z   File "/usr/local/lib/python3.11/dist-packages/django/template/defaulttags.py", line 238, in render
2023-07-04T08:20:06.566049010Z     nodelist.append(node.render_annotated(context))
2023-07-04T08:20:06.566051010Z                     ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
2023-07-04T08:20:06.566052677Z   File "/usr/local/lib/python3.11/dist-packages/django/template/base.py", line 966, in render_annotated
2023-07-04T08:20:06.566054385Z     return self.render(context)
2023-07-04T08:20:06.566056010Z            ^^^^^^^^^^^^^^^^^^^^
2023-07-04T08:20:06.566057593Z   File "/usr/local/lib/python3.11/dist-packages/django/template/defaulttags.py", line 321, in render
2023-07-04T08:20:06.566059510Z     return nodelist.render(context)
2023-07-04T08:20:06.566061343Z            ^^^^^^^^^^^^^^^^^^^^^^^^
2023-07-04T08:20:06.566062927Z   File "/usr/local/lib/python3.11/dist-packages/django/template/base.py", line 1005, in render
2023-07-04T08:20:06.566075885Z     return SafeString("".join([node.render_annotated(context) for node in self]))
2023-07-04T08:20:06.566079385Z                               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
2023-07-04T08:20:06.566081052Z   File "/usr/local/lib/python3.11/dist-packages/django/template/base.py", line 1005, in <listcomp>
2023-07-04T08:20:06.566082843Z     return SafeString("".join([node.render_annotated(context) for node in self]))
2023-07-04T08:20:06.566084593Z                                ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
2023-07-04T08:20:06.566086968Z   File "/usr/local/lib/python3.11/dist-packages/django/template/base.py", line 966, in render_annotated
2023-07-04T08:20:06.566090177Z     return self.render(context)
2023-07-04T08:20:06.566094177Z            ^^^^^^^^^^^^^^^^^^^^
2023-07-04T08:20:06.566095885Z   File "/usr/local/lib/python3.11/dist-packages/django/template/base.py", line 1064, in render
2023-07-04T08:20:06.566097593Z     output = self.filter_expression.resolve(context)
2023-07-04T08:20:06.566099218Z              ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
2023-07-04T08:20:06.566100802Z   File "/usr/local/lib/python3.11/dist-packages/django/template/base.py", line 715, in resolve
2023-07-04T08:20:06.566102593Z     obj = self.var.resolve(context)
2023-07-04T08:20:06.566104468Z           ^^^^^^^^^^^^^^^^^^^^^^^^^
2023-07-04T08:20:06.566106052Z   File "/usr/local/lib/python3.11/dist-packages/django/template/base.py", line 847, in resolve
2023-07-04T08:20:06.566107760Z     value = self._resolve_lookup(context)
2023-07-04T08:20:06.566110010Z             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
2023-07-04T08:20:06.566111718Z   File "/usr/local/lib/python3.11/dist-packages/django/template/base.py", line 914, in _resolve_lookup
2023-07-04T08:20:06.566113468Z     current = current()
2023-07-04T08:20:06.566115260Z               ^^^^^^^^^
2023-07-04T08:20:06.566118552Z   File "/usr/local/lib/python3.11/dist-packages/wagtail/admin/ui/components.py", line 20, in render_html
2023-07-04T08:20:06.566120302Z     return template.render(context_data)
2023-07-04T08:20:06.566121885Z            ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
2023-07-04T08:20:06.566123510Z   File "/usr/local/lib/python3.11/dist-packages/django/template/backends/django.py", line 61, in render
2023-07-04T08:20:06.566125468Z     return self.template.render(context)
2023-07-04T08:20:06.566127593Z            ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
2023-07-04T08:20:06.566129218Z   File "/usr/local/lib/python3.11/dist-packages/django/template/base.py", line 175, in render
2023-07-04T08:20:06.566131093Z     return self._render(context)
2023-07-04T08:20:06.566132927Z            ^^^^^^^^^^^^^^^^^^^^^
2023-07-04T08:20:06.566134552Z   File "/usr/local/lib/python3.11/dist-packages/django/template/base.py", line 167, in _render
2023-07-04T08:20:06.566140385Z     return self.nodelist.render(context)
2023-07-04T08:20:06.566142135Z            ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
2023-07-04T08:20:06.566143843Z   File "/usr/local/lib/python3.11/dist-packages/django/template/base.py", line 1005, in render
2023-07-04T08:20:06.566145718Z     return SafeString("".join([node.render_annotated(context) for node in self]))
2023-07-04T08:20:06.566147552Z                               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
2023-07-04T08:20:06.566149260Z   File "/usr/local/lib/python3.11/dist-packages/django/template/base.py", line 1005, in <listcomp>
2023-07-04T08:20:06.566151052Z     return SafeString("".join([node.render_annotated(context) for node in self]))
2023-07-04T08:20:06.566153177Z                                ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
2023-07-04T08:20:06.566155177Z   File "/usr/local/lib/python3.11/dist-packages/django/template/base.py", line 966, in render_annotated
2023-07-04T08:20:06.566157218Z     return self.render(context)
2023-07-04T08:20:06.566158927Z            ^^^^^^^^^^^^^^^^^^^^
2023-07-04T08:20:06.566163385Z   File "/usr/local/lib/python3.11/dist-packages/django/template/defaulttags.py", line 238, in render
2023-07-04T08:20:06.566166093Z     nodelist.append(node.render_annotated(context))
2023-07-04T08:20:06.566167760Z                     ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
2023-07-04T08:20:06.566169718Z   File "/usr/local/lib/python3.11/dist-packages/django/template/base.py", line 966, in render_annotated
2023-07-04T08:20:06.566171468Z     return self.render(context)
2023-07-04T08:20:06.566173135Z            ^^^^^^^^^^^^^^^^^^^^
2023-07-04T08:20:06.566174968Z   File "/usr/local/lib/python3.11/dist-packages/wagtail/admin/templatetags/wagtailadmin_tags.py", line 1155, in render
2023-07-04T08:20:06.566176760Z     children = self.nodelist.render(context) if self.nodelist else ""
2023-07-04T08:20:06.566178760Z                ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
2023-07-04T08:20:06.566180385Z   File "/usr/local/lib/python3.11/dist-packages/django/template/base.py", line 1005, in render
2023-07-04T08:20:06.566182052Z     return SafeString("".join([node.render_annotated(context) for node in self]))
2023-07-04T08:20:06.566183760Z                               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
2023-07-04T08:20:06.566185552Z   File "/usr/local/lib/python3.11/dist-packages/django/template/base.py", line 1005, in <listcomp>
2023-07-04T08:20:06.566187718Z     return SafeString("".join([node.render_annotated(context) for node in self]))
2023-07-04T08:20:06.566189510Z                                ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
2023-07-04T08:20:06.566191593Z   File "/usr/local/lib/python3.11/dist-packages/django/template/base.py", line 966, in render_annotated
2023-07-04T08:20:06.566199927Z     return self.render(context)
2023-07-04T08:20:06.566202927Z            ^^^^^^^^^^^^^^^^^^^^
2023-07-04T08:20:06.566204635Z   File "/usr/local/lib/python3.11/dist-packages/django/template/library.py", line 237, in render
2023-07-04T08:20:06.566206385Z     output = self.func(*resolved_args, **resolved_kwargs)
2023-07-04T08:20:06.566208010Z              ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
2023-07-04T08:20:06.566209635Z   File "/usr/local/lib/python3.11/dist-packages/wagtail/admin/templatetags/wagtailadmin_tags.py", line 1083, in component
2023-07-04T08:20:06.566211427Z     return obj.render_html(context)
2023-07-04T08:20:06.566213052Z            ^^^^^^^^^^^^^^^^^^^^^^^^
2023-07-04T08:20:06.566216885Z   File "/usr/local/lib/python3.11/dist-packages/wagtail/admin/ui/components.py", line 15, in render_html
2023-07-04T08:20:06.566218718Z     context_data = self.get_context_data(parent_context)
2023-07-04T08:20:06.566220427Z                    ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
2023-07-04T08:20:06.566222302Z   File "/usr/local/lib/python3.11/dist-packages/wagtail/admin/panels/title_field_panel.py", line 60, in get_context_data
2023-07-04T08:20:06.566224052Z     field.widget.attrs.update(**self.get_attrs())
2023-07-04T08:20:06.566225677Z                                 ^^^^^^^^^^^^^^^^^
2023-07-04T08:20:06.566227927Z   File "/usr/local/lib/python3.11/dist-packages/wagtail/admin/panels/title_field_panel.py", line 81, in get_attrs
2023-07-04T08:20:06.566229760Z     targets = [self.get_target_selector(target) for target in panel.targets]
2023-07-04T08:20:06.566231885Z               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
2023-07-04T08:20:06.566233843Z   File "/usr/local/lib/python3.11/dist-packages/wagtail/admin/panels/title_field_panel.py", line 81, in <listcomp>
2023-07-04T08:20:06.566235593Z     targets = [self.get_target_selector(target) for target in panel.targets]
2023-07-04T08:20:06.566237593Z                ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
2023-07-04T08:20:06.566239552Z   File "/usr/local/lib/python3.11/dist-packages/wagtail/admin/panels/title_field_panel.py", line 128, in get_target_selector
2023-07-04T08:20:06.566241510Z     field = self.form[target]
2023-07-04T08:20:06.566243302Z             ^^^^^^^^^^^^^^^^^
2023-07-04T08:20:06.566245385Z   File "/usr/local/lib/python3.11/dist-packages/django/forms/forms.py", line 180, in __getitem__
2023-07-04T08:20:06.566248218Z     raise KeyError(
2023-07-04T08:20:06.566252677Z KeyError: "Key 'slug' not found in 'CountryPageForm'. Choices are: comment_notifications, country, cover_image, description_de, description_en, description_es, description_fr, description_it, description_nl, expire_at, go_live_at, search_description_de, search_description_en, search_description_es, search_description_fr, search_description_it, search_description_nl, seo_title_de, seo_title_en, seo_title_es, seo_title_fr, seo_title_it, seo_title_nl, show_in_menus, slug_de, slug_en, slug_es, slug_fr, slug_it, slug_nl, title."
```

**Comment by Mostaard** [comment]
I'm getting seeing the same errors

**Comment by Wartijn** [comment]
It looks like the `TitleFieldPanel` [that was added in 5.02](https://docs.wagtail.org/en/stable/releases/5.0.2.html#new-features) is what's breaking it. 
As a workaround you can define your own content_panels, instead of extending `Page.content_panels`

That does mean you won't get the default field, so you'll have to add them yourself. 

So if you had:
```
    content_panels = Page.content_panels + [
        # your panels
    ]
```

That should become
```
    content_panels = [
        FieldPanel("title"),
        FieldPanel("slug"),
       # other panels that Page.content_panels provides
       # your panels
    ]
```

**Comment by DiogoMarques29** [comment]
Fixed in v0.14.1
